AppIntegrity/Sources/AppIntegrity/FirebaseAppIntegrityService.swift

```diff
@@ -44,24 +48,28 @@ public final class FirebaseAppIntegrityService: AppIntegrityProvider {
          providerFactory: AppCheckProviderFactory,
          proofOfPossessionProvider: ProofOfPossessionProvider,
          client: NetworkClient,
-         baseURL: URL) {
+         baseURL: URL,
+         jwtGenerator: JWTGenerator) {
         self.client = client
         self.vendor = vendor
         self.baseURL = baseURL
         self.proofOfPossessionProvider = proofOfPossessionProvider
+        self.jwtGenerator = jwtGenerator
     }
 
     public convenience init(
         client: NetworkClient,
```

Comment:
I feel like this property should be called `networkClient` to avoid any confusion 🙂